tsconfig.json

```diff
@@ -0,0 +1,13 @@
+{
+	"compilerOptions": {
+		"module": "umd",
+		"noImplicitAny": true,
+		"removeComments": true,
+		"preserveConstEnums": true,
+		"sourceMap": true,
+		"target": "es5"
+	},
+	"files": [
+		"src/typed-error.ts"
```

Comment:
I could do that (and would in a lot of modules), however this module is only ever intended to be the one small file so I quite like being explicit about that - if you feel strongly I'll change it though